plugin/confluence_code_block.go

```diff
@@ -0,0 +1,40 @@
+package plugin
+
+import (
+	"fmt"
+	"strings"
+
+	md "github.com/JohannesKaufmann/html-to-markdown"
+	"github.com/PuerkitoBio/goquery"
+)
+
+// ConfluenceCodeBlock converts `<ac:structured-macro>` elements
```

Comment:
Can you change the description to something like that?
```suggestion
// ConfluenceCodeBlock converts `<ac:structured-macro>` elements that are used in Atlassian’s Wiki “Confluence”.
// [Contributed by @Skarlso]
```

That makes it a bit clearer that I can't help much when a bug reports for this plugin is submitted😆

Reply:
Oh absolutely, mate. 😊👍